defaults/parameters.yaml

```diff
@@ -68,6 +68,7 @@ files:
   clades: "defaults/clades.tsv"
   emerging_lineages: "defaults/emerging_lineages.tsv"
   mutational_fitness_distance_map: "defaults/mutational_fitness_distance_map.json"
+  sites_to_mask: "defaults/sites_to_mask.txt"
```

Comment:
Echoing my comment on Slack (and Angie's question on #775), I think this parameter should be called something like `sites_to_mask_for_tree_building` or `sites_to_mask_for_tree_inference`, etc. so we all know when in the workflow masking happens.

Whatever we end up calling it, we should also have a brief description in [the "files" section of the configuration guide](https://github.com/nextstrain/ncov/blob/master/docs/src/reference/configuration.md#files), to make the feature more visible to users. I'm happy to add this, once we've settled on a name.

Reply:
I am happy to have this renamed -- you suggestion sounds good to me.